modules/core/test/webgl/classes/index.js

```diff
@@ -31,6 +31,7 @@ import './uniform-buffer-layout.spec';
 import './transform-feedback.spec';
 
 /*
+Spec runner can't create a WebGL 2 context
```

Comment:
We do test classes with both webgl1 and webgl2 contexts, when run in browser (live/headless). In headless-gl we only test webgl1 contexts of course.

As you can see above webgl2 classes like transform-feedback etc all have test cases.

Reply:
Are the browser tests still run with yarn test? I remember it used to open a browser window, but it stopped doing that recently.

Reply:
Ah, my mistake. I was just looking at the node test results.